cost/google/unutilized_ip_addresses/google_unutilized_ip_addresses.pt

```diff
@@ -7,7 +7,7 @@ severity "low"
 category "Cost"
 default_frequency "daily"
 info(
-      version: "2.9",
+      version: "2.10",
```

Comment:
Should we consider scraping recommendations from this "Google Unutilized IP Addresses" policy?

Reply:
Not scraping from this policy until it can provide accurate cost data.